contracts/cw721-proxy/src/execute.rs

```diff
@@ -0,0 +1,77 @@
+use crate::msg::{get_inner, ExecuteMsg, InnerExecuteMsg, InstantiateMsg, MigrateMsg, QueryMsg};
```

Comment:
This cw721-proxy acts as an cw721 adapter module within Abstract SDK: https://docs.abstract.money/3_framework/6_module_types.html#adapters

Happy to go over the different modules and how to build them, as it removes a ton of boilerplate and the module code has already been audited.